app/src/main/res/layout/shop_item_dialog.xml

```diff
@@ -7,29 +7,29 @@
 
 
     <EditText
-        android:id="@+id/editTextName"
+        android:id="@+id/edit_text_name"
         android:layout_width="200dp"
         android:layout_height="wrap_content"
         android:layout_weight="1"
         android:ems="10"
-        android:inputType="textPersonName"
-        android:text="@string/name" />
+        android:hint="@string/name"
+        android:inputType="textPersonName" />
```

Comment:
Cirrus complains about missing autofillHints, its just adding a string that describes the content of the editText (you can probably put the same as for hint)
Since its really simple and doesnt cost anything to add we might as well add them
The same goes for the 2 other EditTexts further down

Reply:
Alright it is added